team_meetings.yml

```diff
@@ -327,6 +327,41 @@ events:
     repeat:
       interval:
         weeks: 1
+      until: 2023-01-01
+  - summary: "SIG Monitoring"
+    begin: 2023-01-06 12:05:00
+    duration:
+      minutes: 50
+    description: |
+      The Special Interest Group (SIG) Monitoring meets on a fortnightly base (alternating with the audit log WG) to discuss the monitoring needs of SCS Operators, Users and Integrators. Together we shape how monitoring and observability within the SCS landscape looks like.
+      
+      Minutes: https://github.com/SovereignCloudStack/minutes/tree/main/sig-monitoring
+      Etherpad: https://input.osb-alliance.de/p/2023-scs-sig-monitoring
+      Matrix channel for SIG related discussion and coordination: https://matrix.to/#/!ToxwzOWTBqSjxRAwuj:matrix.org?via=matrix.org
+      Jitsi server on https://conf.scs.koeln:8443/SCS-Tech
+      Dial-In: +49-221-292772-611
+      Coordinator: Felix Kronlage-Dammers <fkr[at]osb-alliance.com>
+    location: "https://conf.scs.koeln:8443/SCS-Tech"
+    repeat:
+      interval:
+        weeks: 2
+      until: 2023-06-30
+  - summary: "Audit Log Working Group"
```

Comment:
I just had a discussion with @itrich about the Wording in regards to Team, SIG and Working group. See this discussion: https://github.com/SovereignCloudStack/website/pull/454

I would propose to name it:

SIG Monitoring - Audit Logging Breakout" or similar.

Reply:
I thought about that, but at least in my calendar view, I often only see the first few words because width limitations. Hence, I would very much prefer to have the specific thing first.

(The same rule, btw, applies to HTML titles, because screenreaders naturally read linearly and going from more specific to less specific helps people relying on screenreaders with making decisions based on the title more efficiently.)

Reply:
Good Point. Then "Audit Logging Breakout - SIG Monitoring" ?

Reply:
That I can get on board with. Changed.